version: "3.8"

services:
  beat_pulse-db:
    image: postgres:12.3-alpine
    restart: always
    volumes:
      - beat_pulse-volume:/var/lib/postgresql/data/
    environment:
      POSTGRES_DB: "beat_pulse"
      POSTGRES_PASSWORD: "12345"
    ports:
      - 5440:5432
    networks:
      - beat_pulse_network
  beatweb:
   build:
     context: .
   command: bash -c "python manage.py migrate && python manage.py runserver 0.0.0.0:8010"
   container_name: beatweb
   ports:
     - 8010:8010
   networks:
     - beat_pulse_network
   depends_on:
     - beat_pulse-db
   restart: on-failure:3

volumes:
  beat_pulse-volume:

networks:
 beat_pulse_network:
   driver: bridge